theme_kea/views/snippets/s_cover.xml

```diff
@@ -3,23 +3,32 @@
     <template id="s_cover" inherit_id="website.s_cover">
         <!-- Section -->
         <xpath expr="//section" position="attributes">
-            <attribute name="class" add="o_full_screen_height" separator=" "/>
-            <attribute name="data-oe-shape-data">{"shape":"web_editor/Origins/07_002","flip":[]}</attribute>
+            <attribute name="class" add="oe_img_bg o_full_screen_height" remove="parallax s_parallax_is_fixed bg-black-50" separator=" "/>
+            <attribute name="data-scroll-background-ratio">0</attribute>
+            <attribute name="data-oe-shape-data">{"shape":"web_editor/Floats/02","flip":[]}</attribute>
+            <attribute name="style">background-image: url('/web/image/website.s_cover_default_image'); background-position: 50% 0;</attribute>
         </xpath>
-        <!-- Title & Paragraph -->
-        <xpath expr="//div[hasclass('container')]" position="replace">
-            <div class="container s_allow_columns text-center">
-                <h1>
-                    <font style="font-size: 62px;"><b>The Ultimate Laptop</b></font>
-                </h1>
-                <p class="lead">Speed up your work with the last laptop. 3.4Ghz, 8 cores.<br/>
-                You’ll be unstoppable all day long (up to 11 hours).</p>
-                <p><a t-att-href="cta_btn_href" class="btn btn-primary mb-2"><t t-esc="cta_btn_text">Discover</t></a></p>
-            </div>
+        <!-- Update Filter -->
+        <xpath expr="//div[hasclass('o_we_bg_filter')]" position="replace">
+            <div class="o_we_bg_filter" style="background-color: rgba(0, 0, 0, 0.10)"/>
         </xpath>
         <!-- Shape -->
-        <xpath expr="//section//*[last()]" position="after">
-            <div class="o_we_shape o_web_editor_Origins_07_002"/>
+        <xpath expr="//div[hasclass('container')]" position="before">
+            <div class="o_we_shape o_web_editor_Floats_02"/>
+        </xpath>
```

Comment:
Since you replace the filter in the xpath just above, could you not add the shape div at the same time? That would avoid an xpath for it, so better perf, less risk to have it broken at some point, less code, etc.